pkg/deployer/k8s/deployer.go

```diff
@@ -119,6 +119,7 @@ func (d *deployer) CreateGreptimeDBCluster(ctx context.Context, name string, opt
 	if err != nil {
 		return err
 	}
+	d.logger.V(3).Infof("create cluster with values: %v", values)
```

Comment:
suggestion: The log description is `d.logger.V(3).Infof("create greptimedb cluster with values: %v", values)`